bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBS/VVjj_semileptonic/aQGC/aQGC_WMhadZlepJJ_EWK_LO_NPle1_mjj100pt10/aQGC_WMhadZlepJJ_EWK_LO_NPle1_mjj100pt10_run_card.dat

```diff
@@ -0,0 +1,272 @@
+# Control the pt's of leptons sorted by pt                           *
+#*********************************************************************
+ 0.0   = ptl1min ! minimum pt for the leading lepton in pt
+ 0.0   = ptl2min ! minimum pt for the second lepton in pt
+ 0.0   = ptl3min ! minimum pt for the third lepton in pt
+ 0.0   = ptl4min ! minimum pt for the fourth lepton in pt
+ -1.0  = ptl1max ! maximum pt for the leading lepton in pt 
+ -1.0  = ptl2max ! maximum pt for the second lepton in pt
+ -1.0  = ptl3max ! maximum pt for the third lepton in pt
+ -1.0  = ptl4max ! maximum pt for the fourth lepton in pt
+#*********************************************************************
+# Control the Ht(k)=Sum of k leading jets                            *
+#*********************************************************************
+ 0.0   = htjmin ! minimum jet HT=Sum(jet pt)
+ -1.0  = htjmax ! maximum jet HT=Sum(jet pt)
+ 0.0   = ihtmin  !inclusive Ht for all partons (including b)
+ -1.0  = ihtmax  !inclusive Ht for all partons (including b)
+ 0.0   = ht2min ! minimum Ht for the two leading jets
+ 0.0   = ht3min ! minimum Ht for the three leading jets
+ 0.0   = ht4min ! minimum Ht for the four leading jets
+ -1.0  = ht2max ! maximum Ht for the two leading jets
+ -1.0  = ht3max ! maximum Ht for the three leading jets
+ -1.0  = ht4max ! maximum Ht for the four leading jets
+#***********************************************************************
+# Photon-isolation cuts, according to hep-ph/9801442                   *
+# When ptgmin=0, all the other parameters are ignored                  *
+# When ptgmin>0, pta and draj are not going to be used                 *
+#***********************************************************************
+ 0.0 = ptgmin ! Min photon transverse momentum
+ 0.4 = R0gamma ! Radius of isolation code
+ 1.0 = xn ! n parameter of eq.(3.4) in hep-ph/9801442
+ 1.0 = epsgamma ! epsilon_gamma parameter of eq.(3.4) in hep-ph/9801442
+ True = isoEM ! isolate photons from EM energy (photons and leptons)
+#*********************************************************************
+# WBF cuts                                                           *
+#*********************************************************************
+ 0.0   = xetamin ! minimum rapidity for two jets in the WBF case  
+ 0.0   = deltaeta ! minimum rapidity for two jets in the WBF case 
+#*********************************************************************
+# KT DURHAM CUT                                                      *
+#*********************************************************************
+ -1.0    =  ktdurham        
+ 0.4  =  dparameter 
+#*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+ 5 = maxjetflavor    ! Maximum jet pdg code
```

Comment:
The setting is not consistent with your proc_card.
Please check your model file whether your b mass is set to 0 or not (if not, this should be changed to 4).

Sorry for not realizing this in the first review..

Reply:
Hi @sihyunjeon 

To check if my model is 4 flavor or 5 flavour, I did the following:

import model SM_Ltotal_Ind5v2020v2_UFO
generate p p > t t~

this succeeded with 8 diagrams

Then I did 
b b > t t~
But this failed. So, I concluded that my model is 4 flavor model. Am I right?

If yes, then I can update the cards accordingly.

Reply:
What you actually should've done is "b b~" not "b b" ;).

The way you can check your flavour scheme is by doing

import model XXX  
display multiparticles  

Then you would see how your protons are defined.
I checked your model file myself and it's 5f

Reply:
Hi @sihyunjeon 
I have checked with bb~ and it generates one Feynman diagram so this model includes the b quark. Also, I have checked the param card which says that b-quark is assigned zero mass (in fact it assign zero mass to all quarks). 

I am going to add b b~ in proton definition in proc card, if you permit in the next reply.
also, do this ensures that this model handles the b-quark appropriately?

![Screenshot from 2021-08-13 13-07-04](https://user-images.githubusercontent.com/39457794/129321856-fb8830bd-24ee-4bf8-8121-bc587cdd3ad5.png)

Reply:
Hi @sihyunjeon 
Can you please update me further, so that I can proceed further?

Reply:
What is your concern? The calculation will be done in the 5FS. On contribution you are worried about that is missing? 
May I ask on top: customize card, model card and run card are always the same? Just madpin and process are different. 
We can continue the review but why not working with a simple script. If you have a wrong line in any of the run_cards we cannot spot.

Reply:
@agrohsje, Yes we have run, customized, extramodel, and run card same, also I have verified that they are the same. Only madspin and process card are different as per the required process. 

We could have script for it, but I have cloned it from one of my colleagues @ram1123 repository and made made changes as per requirement.

Reply:
Did you test which PDF is picked up? If you run in 5FS but have no b's in p you might prefer 4FS pdfs.